emop-maven-plugin/src/main/java/edu/cornell/AffectedSpecsMojo.java

```diff
@@ -34,11 +49,92 @@ public void execute() throws MojoExecutionException {
         MessageHandler mh = new MessageHandler();
         compiler.run(arguments, mh);
         IMessage[] ms = mh.getMessages(IMessage.WEAVEINFO, false);
+        computeMapFromMessage(ms);
+        writeMapToFile("txt");
         Writer.writeToFile(Arrays.asList(ms), getArtifactsDir() + File.separator + "join-points");
         getLog().info("[eMOP] Number of impacted classes: " + getImpacted().size());
         getLog().info("[eMOP] Number of messages to process: " + Arrays.asList(ms).size());
     }
 
+    /**
+     * Compute a mapping from affected classes to specifications based on the messages from AJC.
+     * TODO: Handling output messages are definitely slower than reading them directly from AJC
```

Comment:
No worries for now, we will eventually get this directly from inside AJC.

How much slower is it?

Reply:
Okay.